addons/web_unsplash/static/src/js/unsplash_image_widget.js

```diff
@@ -198,15 +172,10 @@ widgetsMedia.ImageWidget.include({
         this.$('#appIdInput').toggleClass('is-invalid', !appId);
 
         if (key && appId) {
-            var params = {
-                'key': key,
-                'appId': appId,
-            };
-
             if (!this.$el.find('.is-invalid').length) {
                 this._rpc({
                     route: '/web_unsplash/save_unsplash',
-                    params: params,
+                    params: {key, appId},
```

Comment:
Still do not like this form (but that's my problem), but here I don't like it for two reasons:
- Appears like an innocent JS variable but is in fact used as a python variable name (so you must never rename that innoncent JS variable here) without changing the python.
- Was like that before but why would a python parameter be camelCased, especially when it was not coded by me but by @rdeodoo :smile:

Reply:
still going to inline it rather than declaring a variable that's never going to be modified. Would you like me to camel case appId here and in Python?

Reply:
For me, it should have been snake_cased at the beginning but no point in changing that. Just make an explicit mapping Python key -> JS value. Going through temporary variable or not.